apps/bfd-model/bfd-model-rif-samples/dev/synthea_releases/01-20-2022/characteristics.csv

```diff
@@ -0,0 +1,10001 @@
+-10000000019401, 1S00E00FT01           , ,10,10,0,0,1,6,0,4
+-10000000019273, 1S00E00FQ73           , Z1000,9,9,0,1,1,8,0,12
+-10000000010263, 1S00E00AD63           , Z1001,10,11,0,1,3,22,0,36
+-10000000010264, 1S00E00AD64           , ,9,0,0,1,1,11,0,2
+-10000000010265, 1S00E00AD65           , Z1001,9,1,0,1,18,264,0,362
+-10000000010266, 1S00E00AD66           ," Z1003,Z1003",10,11,0,0,1,6,1,0
+-10000000010267, 1S00E00AD67           , Z1002,10,0,0,0,0,16,0,15
+-10000000010268, 1S00E00AD68           , ,10,11,0,0,2,4,1,7
+-10000000010269, 1S00E00AD69           , Z1002,10,11,0,0,2,5,2,3
+-10000000010270, 1S00E00AD70           , ,10,0,0,0,1,6,0,0
+-10000000010271, 1S00E00AD71           ," Z1001,Z1001",10,1,16,0,0,7,0,44
+-10000000010273, 1S00E00AD73           ," Z1007,Z1007",10,11,0,1,11,35,0,2
+-10000000010274, 1S00E00AD74           , Z1007,9,10,0,1,0,6,0,3
+-10000000010275, 1S00E00AD75           , Z1003,10,10,0,0,0,19,1,40
+-10000000018582, 1S00E00FG82           , ,10,11,0,0,0,6,0,0
+-10000000010163, 1S00E00AC63           , Z1000,10,11,0,0,1,9,0,21
+-10000000010164, 1S00E00AC64           , ,10,0,0,0,2,21,1,0
+-10000000010165, 1S00E00AC65           , Z1008,10,12,0,0,1,18,1,11
+-10000000010166, 1S00E00AC66           , ,10,0,29,0,0,10,0,0
+-10000000010169, 1S00E00AC69           , Z1002,9,10,0,0,1,8,0,2
+-10000000010170, 1S00E00AC70           , Z1000,10,10,0,0,3,13,2,34
+-10000000010174, 1S00E00AC74           , Z1006,10,0,0,1,0,23,0,14
+-10000000010175, 1S00E00AC75           , Z1002,0,0,0,0,0,6,1,1
+-10000000010176, 1S00E00AC76           , Z1005,10,11,0,1,1,16,1,35
+-10000000010177, 1S00E00AC77           , ,10,0,0,0,1,5,0,1
+-10000000010181, 1S00E00AC81           , ,10,0,0,1,3,27,0,66
+-10000000010182, 1S00E00AC82           , Z1006,10,0,0,1,2,8,1,10
+-10000000010183, 1S00E00AC83           , ,10,11,0,0,1,7,0,3
+-10000000010184, 1S00E00AC84           , ,9,11,0,0,0,20,1,11
+-10000000010185, 1S00E00AC85           , Z1000,10,12,0,0,1,11,0,2
+-10000000019305, 1S00E00FR05           , Z1002,10,10,0,0,1,6,0,13
+-10000000019348, 1S00E00FR48           ," Z1003,Z1003",9,9,0,0,3,11,0,24
+-10000000010861, 1S00E00AK61           , ,10,0,6,0,1,7,0,1
+-10000000010863, 1S00E00AK63           , ,10,10,0,1,1,29,0,5
+-10000000010864, 1S00E00AK64           , Z1009,10,0,0,0,1,7,1,0
+-10000000019349, 1S00E00FR49           , ,10,0,0,0,4,147,0,92
+-10000000010865, 1S00E00AK65           , Z1005,10,0,22,0,1,7,0,1
+-10000000010866, 1S00E00AK66           , Z1005,10,0,0,1,0,3,0,10
+-10000000010869, 1S00E00AK69           , ,10,0,0,0,7,49,0,5
+-10000000010870, 1S00E00AK70           , Z1003,10,10,20,0,0,5,0,10
+-10000000010871, 1S00E00AK71           , Z1009,10,12,0,1,1,20,0,4
+-10000000010874, 1S00E00AK74           , Z1004,10,0,0,0,2,16,0,13
+-10000000010875, 1S00E00AK75           , ,10,10,0,0,0,4,0,0
+-10000000010876, 1S00E00AK76           , Z1004,10,0,0,1,0,6,0,0
+-10000000010877, 1S00E00AK77           , Z1000,10,0,0,0,1,7,0,21
+-10000000
```

Comment:
this file is not consistent with the one from last time. it is not sorted by the first column and has a lot of extra whitespace.